spinalcordtoolbox/scripts/sct_image.py

```diff
@@ -174,6 +184,12 @@ def main(argv=None):
 
     fname_in = arguments.i
     n_in = len(fname_in)
+    # TODO: The functions for '-concat', '-omc', and '-display-warp' take in filenames, so they ignore 'im_in'.
+    #  Instead, we should harmonize this functionality so that 'im_in' is used everywhere.
+    im_in = Image(fname_in[0])
```

Comment:
While finishing up this PR, I made this observation.

This is a relatively simple fix, but I don't want the scope of this PR to grow any further, so I'll follow up in a separate PR.

Reply:
I made an issue for this here: https://github.com/neuropoly/spinalcordtoolbox/issues/3205